app/validators/blocks/calculated_summary_block_validator.py

```diff
@@ -20,31 +20,40 @@ def validate(self):
         if (answers := self.get_answers(self.answers_to_calculate)) is None:
             return self.errors
 
-        self.validate_single_answer_is_for_dynamic_answers(answers)
+        self.validate_single_answer_is_for_repeating_answers(answers)
         self.validate_answer_id_set_before_calculated_summary_block()
         self.validate_answer_id_for_calculated_summary_not_in_different_section()
 
         self.validate_answer_types(answers)
 
         return self.errors
 
-    def validate_single_answer_is_for_dynamic_answers(self, answers: list[dict]):
-        """Validate that if there is only one answer in the answers_to_calculate list, it's for dynamic answers"""
+    def validate_single_answer_is_for_repeating_answers(self, answers: list[dict]):
+        """Validate that if there is only one answer in the answers_to_calculate list, it's for repeating answers"""
         if len(answers) == 1:
             single_answer_id = answers[0]["id"]
+            # check if its dynamic
             if (
                 single_answer_id
-                not in self.questionnaire_schema.get_all_dynamic_answer_ids(
+                in self.questionnaire_schema.get_all_dynamic_answer_ids(
                     self.questionnaire_schema.get_block_id_by_answer_id(
                         single_answer_id
                     )
                 )
             ):
-                self.add_error(
-                    self.CALCULATED_SUMMARY_WITH_NON_DYNAMIC_SINGLE_ANSWER,
-                    block_id=self.block["id"],
-                    answer_id=single_answer_id,
-                )
+                return
+            # check if it's for a repeating question
+            question_block = self.questionnaire_schema.get_block_by_answer_id(
+                single_answer_id
+            )
+            if question_block["type"] == "ListRepeatingQuestion":
+                return
+
+            self.add_error(
+                self.CALCULATED_SUMMARY_WITH_NON_DYNAMIC_SINGLE_ANSWER,
```

Comment:
Worth changing the var name from dynamic to repeating answer then?.

Reply:
Yep, agree 👍